tests/create_tests_of_examples.py

```diff
@@ -10,22 +10,37 @@
 
 download_assets()
 
+EXAMPLES_TO_SKIP = [
+    "action_primitives.rs_int_example",
+    "action_primitives.solve_simple_task",
+    "action_primitives.wip_solve_behavior_task",
+    "learning.navigation_policy_demo",
+    "teleoperation.robot_teleoperate_demo",
+    # TODO: Temporarily skip the following examples
+    # "object_states.attachment_demo",  # seg fualt??
+    # "environments.behavior_env_demo",  # This only works with pre-sampled cached BEHAVIOR activity scene
```

Comment:
does behavior_env_demo break with the `develop` branch of `BDDL` for `online_sampline=True`?

Reply:
Unfortunately yes, there seem to be some bddl/asset problems here:
```
  File "/home/yhang/OmniGibson/omnigibson/tasks/behavior_task.py", line 325, in initialize_activity
    accept_scene, feedback = self.sampler.sample()
  File "/home/yhang/OmniGibson/omnigibson/utils/bddl_utils.py", line 612, in sample
    accept_scene, feedback = self._prepare_scene_for_sampling()
  File "/home/yhang/OmniGibson/omnigibson/utils/bddl_utils.py", line 684, in _prepare_scene_for_sampling
    error_msg = self._build_inroom_object_scope()
  File "/home/yhang/OmniGibson/omnigibson/utils/bddl_utils.py", line 920, in _build_inroom_object_scope
    valid_models = {
  File "/home/yhang/OmniGibson/omnigibson/utils/bddl_utils.py", line 922, in <dictcomp>
    get_all_object_category_models_with_abilities(
  File "/home/yhang/OmniGibson/omnigibson/utils/asset_utils.py", line 290, in get_all_object_category_models_with_abilities
    if supports_abilities(abilities_info, prim):
  File "/home/yhang/OmniGibson/omnigibson/utils/asset_utils.py", line 273, in supports_abilities
    if not requirement.is_compatible_asset(prim=obj_prim)[0]:
  File "/home/yhang/OmniGibson/omnigibson/object_states/object_state_base.py", line 53, in is_compatible_asset
    raise NotImplementedError
NotImplementedError
Uncaught exception. Entering post mortem debugging
Running 'cont' or 'step' will restart the program
> /home/yhang/OmniGibson/omnigibson/object_states/object_state_base.py(53)is_compatible_asset()
-> raise NotImplementedError
```
`commercial_kitchen_sink : nmxgbm` is missing `ParticleRequirement`